### PR TITLE
Removed Quickfacts &  PSA's .sla and PDF's files, left only .md files

### PR DESCRIPTION
**Pull request by lh1008**
Hi @erciccione,

Adding the **'zh' and 'ru'** folders that were missing the .md files.

In the 'zh' folders added the **giant_zh_.md** files and also changed the folder to _/storyboard_. Noticed that in the #85 PR I approved it and did not notice the .sla and pdf's files were removed and those files are not ment to be removed yet, so I'm adding the files again for anyone who is still looking at the files. 

Thank you guys :)

### CODE REVIEW

**Comment by lh1008** [comment]
I just noticed that the .sla files were deleted. To err is of humans, forgive my mistake.

In #77 I posted those files were going to be deleted after we finish all the rest of the .md files and @BlackLotus64, not sure why, he deleted those files, and he is also a human, and is ok to have mistakes, I will be speaking with him also. 

I will be removing the rest of the .sla files. 

Thank you for the notice.

**Comment by lh1008** [comment]
Hey @erciccione,

I removed all the .sla files and PDF's. I'm currently working on the final design edits for the PDF's so we can add them to the repository. Now we can work on a clean repo and with a smaller size :).

The only folder that has all the documents and I left them intact is the `en` folder. 

Added new instructions on the README files also.

Thanks

File: monero-outreach-docs/translations/ru/PSA/files/wallet/wallet_ru.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  b. Руководство по безопасному созданию холодного кошелька
+r/Monero/comments/48cgmd/an_extensive_guide_for_securely_generating_an
+
+3. Подумайте над тем, чтобы держать холодный кошелек для XMR, которые вы планируете хранить какое-то время, и горячий кошелек для XMR, которые вы собираетесь потратить или продать. Рассуждать следует так: Холодный кошелек = Сберегательный счет и Горячий кошелек = Расчетный счет.
+
+4. Посетите онлайн форумы и выясните различные способы безопасного и защищенного хранения приватных ключей и мнемонической фразы, затем выберите тот способ, который лучше всего подходит для вас. Начать стоит здесь: https://www.getmonero.org/ru/community/hangouts/. Уже заметили, что исследования это большая тема?
+
+5. Следуйте другим лучшим практикам таким, как: никогда никому не показывайте ваши приватные ключи, используйте обменники только с хорошей репутацией, обновляйте ваш кошелек (кошельки) после каждого обновления/hard fork, никогда никому не говорите сколько у Вас XMR и всегда стремитесь изучить как можно больше!
+
```
Comment (erciccione) [inline monero-outreach-docs/translations/ru/PSA/files/wallet/wallet_ru.md]: All the following newlines are not necessary

**Comment by lh1008** [comment]
Yeah, all the images are going to be removed and the PDF's are those left in each folder so anyone can download. That way we reduce the repo size to the minimum.
Not sure if you remember @thunderosa said github is not a good platform to manage images due to size issues.

**Pull request merged**